src/db/LeafDB.cpp

```diff
@@ -0,0 +1,49 @@
+#include <mw/db/LeafDB.h>
+#include "common/Database.h"
+#include "common/SerializableVec.h"
+
+static const DBTable LEAF_TABLE = { 'L' };
```

Comment:
I was thinking more along the lines of a unique char for each type, like 'O' for outputs, 'K' for kernels, 'R' for rangeproofs. It's probably safe to share the same prefix, but I don't think it will lead to best performance for us when pruning, resyncing, etc.